doc/docs/master/enterprise/auth/auth-config.md

```diff
@@ -0,0 +1,73 @@
+# Manage Authentication Configuration
+
+The Pachyderm authentication configuration file includes 
+information about the identity provider (IdP) configured in Pachyderm.
+The file is stored in the Pachyderm etcd container, and you
+can access it by using the following commands:
+
+* To view the auth configuration, run:
+
+  ```bash
+  pachctl auth get-config
+  ```
+
+* To edit the auth configuration, run:
+
+  ```bash
+  pachctl auth set-config <<EOF
```

Comment:
maybe using the `-f` flag to use a json file is more intuitive?

Reply:
Great! Thanks for the tip! Added.